modules/ROOT/pages/apikit-workflow-convert-existing.adoc

```diff
@@ -1,104 +1,239 @@
+		<db:select config-ref="Database_Config" >
+			<db:sql ><![CDATA[SELECT * FROM USERS WHERE :userIdParameter == userID]]></db:sql>
+			<db:input-parameters ><![CDATA[#[{userIdParameter: attributes.queryParams.id}]]]></db:input-parameters>
+		</db:select>
+	</flow>
+</mule>
+----
+
+== Manual Approach
+
+You can follow this workflow to configure a functional router for your existing flows manually.
+
+=== Step 1 Design Your API Specification
+
+To define the API specification for your service, see xref:apikit-4-implement-rest-api.adoc#design-your-api-specification[Design Your API Specification at Building, Implementing, and Testing a REST API]. To illustrate this example, create or copy the API specification and JSON example used in that workflow to the `/src/main/resources/api` folder.
+
+=== Step 2: Create a Main Flow
+
+Within the APIkit project, construct a main flow that serves as the entry point.
+
+. Drag the following components from the Mule palette to the canvas:
 +
-Ignore error messages in the Problems window that pertain to undefined attributes.
-
-== Create a Main Flow
-
-Within the APIkit project, build a flow.
-
-. Drag the following components from the Mule palette:
+* An *HTTP Listener* as a flow source
+* An *APIkit Router* operation within the flow process
 +
-* An HTTP Listener
-* An APIkit Router
+. In the properties editor for the *HTTP Listener*, in *Connector Configuration*, select the existing `HTTP_Listener_Config`.
+. Specify a listener path for the APIkit router. Enter `/api/*` in `Path` field.
+. In the properties editor for the APIkit router, in *Router Configuration*, click `Add` to create a new configuration.
 +
-. In the properties editor for APIkit Router, in Router Configuration, select `api-config`.
-. In the properties editor for HTTP Listener, in Connector Configuration, select api-httpListenerConfig, and set a path. For example, in Path, enter `/`.
-
-
-== Generate REST Backend Flows
-
-When you create a RAML in Studio, generating backend flows is optional. Error handling is generated for a RAML-based API. You need to reference generated flows from the main flow. This procedure assumes you have already created a main flow.
-
-To generate backend flows:
-
-. In the Package Explorer, right-click the project name.
-. Select Mule > Generate Flows From REST API.
-+
-The backend flows appear below the main flow.
-
-== Map RAML Resources to Actions
-
-As part of the backend-first design process, you map RAML resources to actions as shown in the following procedure. This procedure assumes you have generated backend flows.
-
-To map resources to actions:
-
-. Click the APIkit Router to open the *Properties Editor*. In *Router configuration*, click image:add-16x16.png["Add button."].
+The *Global Element Properties* wizard appears.
+. In *API Definition*, select the path `api/api.raml`.
+. Accept the configuration.
+. Delete the HTTP listener from `GET-Users-FLOW`.
```

Comment:
If you are still talking about the MuleSoft HTTP Listener source, change "the HTTP listener" to "*HTTP Listener*

Reply:
OK! Thanks!